### PR TITLE
Categorize admin settings sidebar panel

### PR DESCRIPTION
**Pull request by HesterG**
This PR reorganize and categorize the admin settings sidebar panel into groups:

- User Accounts, User Emails, Organizations, Authentication Sources -> Identity & Access
- Repositories, Packages -> Code Assets
- Webhooks, Applications -> Integrations

Before:

Screen Shot 2023-07-21 at 10 30 28

After:

all configurable settings enabled (package, oauth, webhook)

Screen Shot 2023-07-21 at 10 27 30

all configurable settings disabled (package, oauth, webhook)

Screen Shot 2023-07-21 at 10 25 19

only oauth enabled

Screen Shot 2023-07-21 at 10 26 23

only webhook enabled

Screen Shot 2023-07-21 at 10 26 55

### CODE REVIEW

**Comment by wxiaoguang** [comment]
"Organization Structure" seems pretty longer than others .....

**Comment by HesterG** [comment]
> "Organization Structure" seems pretty longer than others .....

Is there a suggestion for this name? It is actually almost the same length as "Authentication Sources"

**Comment by harryzcy** [comment]
@HesterG Maybe call it "Identity & Access"? just a little bit shorter.

**Comment by HesterG** [comment]
> @HesterG Maybe call it "Identity & Access"? just a little bit shorter.

Sounds OK. I will wait for some other suggestion, if there is no incoming suggestion, i will go with this one.

**Comment by HesterG** [comment]
Updated "Organization Structure" to "Identity & Access"

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -2627,10 +2627,13 @@ teams.invite.description = Please click the button below to join the team.
 
 [admin]
 dashboard = Dashboard
+indetity_access = Identity & Access
```
Comment (delvh) [inline options/locale/locale_en-US.ini]: I think you mean `identity_access`?

File: templates/admin/navbar.tmpl [hunk 2]
```diff
@@ -4,35 +4,60 @@
...
-			{{.locale.Tr "admin.repositories"}}
-		</a>
-		{{if .EnablePackages}}
-			<a class="{{if .PageIsAdminPackages}}active {{end}}item" href="{{AppSubUrl}}/admin/packages">
-				{{.locale.Tr "packages.title"}}
-			</a>
-		{{end}}
-		{{if not DisableWebhooks}}
+		<details class="item toggleable-item" {{if or .PageIsAdminUsers .PageIsAdminEmails .PageIsAdminOrganizations .PageIsAdminAuthentications}}open{{end}}>
+			<summary>{{.locale.Tr "admin.indetity_access"}}</summary>
```
Comment (delvh) [inline templates/admin/navbar.tmpl]:
```suggestion
			<summary>{{.locale.Tr "admin.identity_access"}}</summary>
```

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (delvh) [reply options/locale/locale_en-US.ini]:
```suggestion
identity_access = Identity & Access
```

**Review by delvh — APPROVED** [review]

**Review by puni9869 — APPROVED** [review]

**Comment by puni9869** [comment]
Not a blocker, could you arrange them alphabetically.

**Comment by HesterG** [comment]
> Not a blocker, could you arrange them alphabetically [the inner list items].

Updated https://github.com/go-gitea/gitea/pull/26030/commits/c561d3c8eee11711510659354dbd5fdcee6aa31c

**Pull request merged**